### PR TITLE
docs(modal): add demo for modal window with one tooltip and one popover in the body of the modal

### LINKED ISSUE #3817: docs(modal): add demo for modal window with one tooltip and one popover in the body of the modal
Need to add demo for modal window with one tooltip  and one popover in the body of the modal  ( as an example) .

related to #3258

### PR DESCRIPTION
**Pull request by ghost**
The demo for modal window was added with:
  - one tooltip 
  - one popover 
 in the body of the modal ( as an example) . 
You can see it in `Modal window with tooltip and popover` section
joxi_screenshot_1521196787334

closes #3817

### CODE REVIEW

File: demo/src/app/components/+modal/modal-section.list.ts [hunk 1]
```diff
@@ -171,6 +172,14 @@ export const demoComponentContent: ContentSection[] = [
           want to defer that until user actually sees modal content. I.e. for a "Select e-mail recipient" modal
           you might want to defer recipient list loading until the modal is shown.</p>`,
         outlet: DemoAutoShownModalComponent
+      },
+      {
+        title: 'Modal window with other components',
+        anchor: 'modal-with-popups',
+        component: require('!!raw-loader?lang=typescript!./demos/modal-with-popups/modal-with-popups.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/modal-with-popups/modal-with-popups.html'),
+        description: `<p>Add other components to modal window (<code>popover</code> and <code>tooltip</code> in this case)</p>`,
```
Comment (IlyaSurmay) [inline demo/src/app/components/+modal/modal-section.list.ts]: Modals work fine with other components. In this example, there are popover and tooltip in the modal.

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by ghost** [comment]
@YevheniiaMazur please don't check this pr, while development isn't fixed

**Comment by ghost** [comment]
I've changed buttons_page_spec.ts to run tests (only add missing bracket )

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by ghost** [comment]
@YevheniiaMazur updated my branch. this pr could be checked.

**Comment by YevheniiaMazur** [comment]
Needs to fix conflicting files

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
@svetoldo4444ka , please, update related to this PR issue
Also, please, update demo name and description for making it more clear due to this moments:
1) Modal window contains header and body and both can contain other components
2) Now demo name is modal-with-popups, but it contains only one popup and one tooltip, please, make this name closer to content
demomodalnamenotclear
Otherwise, looks good

**Comment by ghost** [comment]
@YevheniiaMazur @IlyaSurmay @EvilAlexei
It was decided:
- change `closes` word to `related` in the issue for this pr (
joxi_screenshot_1521198447206 )
- to add more information in the description section in the issue and pr 
joxi_screenshot_1521198975952
- change section's name and description in the demo 
joxi_screenshot_1521199016948
- not to change name of templateUrl
joxi_screenshot_1521199439751
because it has to be simple and short

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
@svetoldo4444ka , please, use special words, described in GIT documentation for linking issue to PR
https://help.github.com/articles/closing-issues-using-keywords/

**Comment by ghost** [comment]
@YevheniiaMazur fixed

**Comment by YevheniiaMazur** [comment]
Why words `tooltips` and `popovers` in first sentence are marked as code, and at second are not? In both case it is not part of code, only elements' names. And, if I'm not wrong, with word `any` plurals are not used
notcodemarkedascode
Otherwise, looks good

**Comment by ghost** [comment]
@YevheniiaMazur @IlyaSurmay @EvilAlexei 
was decided to mark any tooltips and popovers in the description section as code (because  tooltip and popover name of component)
joxi_screenshot_1521637995268
Description section was taken from bootstrap
joxi_screenshot_1521638115377

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Pull request merged**